### ISSUE TITLE
Accessibility category

### ISSUE THREAD

**Issue by matatk** [mozilla/addons#1226]
I'm not entirely sure if this is the right place (this isn't a bug report, but a request). Please let me know if I should file this somewhere else, such as BugZilla.

I maintain an [extension called "Landmarks"](https://addons.mozilla.org/en-GB/firefox/addon/landmarks/) that is both an assistive technology (it helps keyboard-only users navigate around a page using landmark regions) and a development tool (it helps developers visualise the landmark structure on their pages). I really want to indicate this dual purpose to users—and AMO lets us pick two categories for classifying extensions—but there are two problems:

* There is no 'Accessibility' category in AMO. Whilst accessibility should always be a concern to any type of extension, I feel there should be a category specifically for assistive technologies—i.e. extensions that help people with specific access needs. Other extensions that I think of as assistive technologies are: [a11y-outline](https://addons.mozilla.org/en-GB/firefox/addon/a11y-outline/); [HeadingsMap](https://addons.mozilla.org/en-GB/firefox/addon/headingsmap/?src=recommended); [Tridactyl](https://addons.mozilla.org/en-US/firefox/addon/tridactyl-vim/?src=external-github) (as well as being in Tabs, Search and other categories too perhaps) and other extensions that provide features for changing the colours/colour temperature on-screen, for example. If it's of relevance: other browsers' extension repositories have explicit 'Accessibility' categories.
* Given that there is currently no 'Accessibility' category, I have chosen 'Other' as the category for the Landmarks extension. However, when you select the 'Other' category during the submission process, you can't also pick any of the default categories, so it's not possible to also select 'Web development'.

I have two suggestions:

1. Please provide an 'Accessibility' category on AMO.
2. Please allow us to select one of the default categories in addition to 'Other'.

What about an alternative?

Perhaps you feel that assistive technologies should be included in whichever of the existing categories is the best fit, rather than having a separate category for assistive technologies. In that case, you could add a category called e.g. 'Navigation' that is about helping _anyone_ navigate about the web. Then the various extensions relating to tab management and keyboard navigation above, Landmarks included, could go in that category?

This has some downsides, though...

* You've already got a 'Tabs' category, so there would be overlap.
* I can think of assistive technologies (e.g. colour-/font-related) that wouldn't fit into a category called 'Navigation', so it feels like there may be _another_ missing (or they would just have to pick 'Other' and be less discoverable).
* Also I feel that if I was searching for something that I need to help me access and engage with the web, having a dedicated 'Accessibility' category for assistive technologies would be the most direct way for me to find it.

Thanks for your time :-).

**Comment by jvillalobos** [comment]
Thanks for you feedback. We will be looking into updating our categories probably later this year, so we'll take this into account. I'll be updating https://github.com/mozilla/addons-server/issues/5989# when that happens, so you can follow that issue for updates. I'm closing this as a duplicate.
